### PR TITLE
Tutorial added on cec2013 (+ autopep8 run)

### PR DESCRIPTION
**Pull request by darioizzo**

### CODE REVIEW

**Comment by darioizzo** [comment]
THis has the PR #16 already in merged, so should be merged after

File: doc/sphinx/docs/python/tutorials/cec2013_comp.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
+.. _py_tutorial_cec2013_copm:
+
+Participating to the CEC2013 Competition
+===============================================
+
+In this tutorial we will show how to use pygmo to run algorithms on the test problem suite used in the
+Special Session & Competition on Real-Parameter Single Objective Optimization at CEC-2013, Cancun, Mexico 21-23 June 2013
```
Comment (bluescarni) [inline doc/sphinx/docs/python/tutorials/cec2013_comp.rst]: Looks like a period ``.`` is missing here at the end.

File: include/pagmo/population.hpp [hunk 2]
```diff
@@ -88,7 +88,7 @@ class population
      * Constructs an empty population with a pagmo::null_problem.
      * The random seed is initialised to zero.
      */
-    population() : m_prob(null_problem{}), m_e(0u), m_seed(0u)
+    population() : m_prob(null_problem{}), m_ID(), m_x(), m_f(), m_champion_x(), m_champion_f(), m_e(0u), m_seed(0u)
```
Comment (bluescarni) [inline include/pagmo/population.hpp]: Did these extra inits slip in from the other branch?

File: include/pagmo/population.hpp [hunk 3]
```diff
@@ -108,7 +108,7 @@ class population
      */
     template <typename T, generic_ctor_enabler<T> = 0>
     explicit population(T &&x, size_type pop_size = 0u, unsigned int seed = pagmo::random_device::next())
-        : m_prob(std::forward<T>(x)), m_e(seed), m_seed(seed)
+        : m_prob(std::forward<T>(x)), m_ID(), m_x(), m_f(), m_champion_x(), m_champion_f(), m_e(seed), m_seed(seed)
```
Comment (bluescarni) [inline include/pagmo/population.hpp]: Ditto.

File: pygmo/core.cpp [hunk 4]
```diff
@@ -246,6 +246,17 @@ static inline void population_prob_init(bp::class_<population> &pop_class)
         .def(bp::init<const Prob &, population::size_type, unsigned>());
 }
 
+// champion_x()
```
Comment (bluescarni) [inline pygmo/core.cpp]: These have been eliminated in the other branch, right?

Comment (darioizzo) [inline pygmo/core.cpp]: yes

File: include/pagmo/population.hpp (hunk 3 above)
Comment (darioizzo) [inline include/pagmo/population.hpp]: yes

File: include/pagmo/population.hpp (hunk 2 above)
Comment (darioizzo) [inline include/pagmo/population.hpp]: yes

Comment (darioizzo) [inline include/pagmo/population.hpp]: they will disappear when this gets merged

Comment (bluescarni) [inline include/pagmo/population.hpp]: you sure they will disappear?

Comment (bluescarni) [inline include/pagmo/population.hpp]: I just checked on a local branch, and they did not go away.

I ended up with 2 ``population_test_case`` tests in ``test.py`` as well.

**Comment by darioizzo** [comment]
Not sure what happened here, but as we said it not a good idea to have PRs that interdepend on each other .... my bad

**Pull request merged**